generate-sandpack-config.mjs

```diff
@@ -0,0 +1,118 @@
+import _ from 'lodash';
+
+import fs from 'fs';
+import path from 'path';
+
+const examples = ['banner'];
```

Comment:
Could this be determined without hardcoding? maybe everything in src is considered an example?

Reply:
i ideally want to move all examples under /lib/ or something similar which would make this easier